File: problem-5/problem-5.go
package main

import "fmt"

func main() {
	primes := [...]int{2, 3, 5, 7, 11, 13, 17, 19}
	product := 1

	for i := range len(primes) {
		prime := primes[i]
		subproduct := prime

		for subproduct < 20 {
			product *= prime
			subproduct *= prime
		}
	}

	fmt.Println(product)
}

File: problem-5/problem-5.ts
let primes = [2,3,5,7,11,13,17,19]
let product = 1

for (const prime of primes) {
    let subproduct = prime

    while (subproduct < 20) {
        product *= prime
        subproduct *= prime
    }
}

console.log(product)